sdf/final-project/business.md

```diff
@@ -1,56 +1,49 @@
 # SDF Final Project Rubric - Business
-
 This is intended to be filled out by staff members of DPI Tech Apprenticeship program.
 
-Date/Time:
-
-Trainee Name:
-
-Project Name:
-
-Reviewer Name:
+- Date/Time:
+- Trainee Name:
+- Project Name:
+- Reviewer Name:
 
-## Requirements
+## Functional Requirements Document
 Should have a functional specification document with:
-
-    [ ] a description of the problem
-
-    [ ] proposed solution
-
-    [ ] user stories
-
-    [ ] user interface / flow sketches
-
-[ ] Does the app satisfy the functional requirements?
+- [ ] description of the problem
+- [ ] proposed solution
+- [ ] user stories
+- [ ] domain model, associations
+- [ ] user interface, flow, sketches, pages
 
 ### Score (/5):
 
 ### Notes:
 
-## Landing Page
-[ ] Does the landing page clearly articulate who the app is for and what problem it solves?
-
-[ ] Should have a call to action (sign up, get started, etc.)
-
-[ ] Has a video and/or how it works section.
-
-[ ] Should have a professional/appealing look and feel
+## Satisfy Functional Requirements
+How well does the deployed app satisfy the functional requirements?
+- [ ] 0 - did not deploy or does not satisfy any of the functional requirements
+- [ ] 1-2 - some requirements satisfied, but major gaps and/or bugs
+- [ ] 3-4 - most requirements satisfied, but still more work needed
+- [ ] 5 - 100% of functional requirements satisfied
```

Comment:
I don't quite understand this `Satisfy Functional Requirements` ;  seems redundant with the previous rubric `Functional Requirements Document`?

Reply:
i'm thinking 1 is to just have a functional requirements document and the other is the degree to which your deployed app satisfies it.